main.go

```diff
@@ -100,6 +100,8 @@ const (
 	middlewareTimeout
 )
 
+var enableLogs = true
```

Comment:
Consider removing this global variable and group it under `logsConfig`. You can further enable this flag in `logsConfig` if at least one `{read,write,tail}Endpoint}` is provided.

Reply:
Done